parser.go

```diff
@@ -526,6 +526,19 @@ func (p *Parser) parseOption(s *parseState, name string, option *Option, canarg
 		}
 
 		err = option.Set(nil)
+	} else if option.isTerminated() {
+		args := make([]string, 0)
```

Comment:
`var args []string` (starting with a nil slice instead of making a zero-length slice) is a bit more typical in Go.